apps/web/src/features/no-fee-november/components/NoFeeNovemberTransactionCard/index.tsx

```diff
@@ -0,0 +1,106 @@
+import InfoIcon from '@mui/icons-material/Info'
+
+const NoFeeNovemberTransactionCard = () => {
+  const { isEligible, isLoading, error } = useNoFeeNovemberEligibility()
+
+  // Loading state - show skeleton
+  if (isLoading) {
+    return (
+      <Card className={css.card}>
+        <Stack direction="row" alignItems="center" spacing={3}>
+          <Box className={css.skeletonIcon} />
+          <Box flex={1}>
+            <Box className={`${css.skeletonBox} ${css.skeletonTitle}`} />
+            <Box className={`${css.skeletonBox} ${css.skeletonSubtitle}`} />
+          </Box>
+        </Stack>
+      </Card>
+    )
+  }
+
+  // Error state - show eligible content (fail gracefully)
+  if (error) {
+    return (
+      <Card className={css.card}>
+        <Stack direction="row" alignItems="center" spacing={3}>
+          <Box className={css.iconContainer}>
+            <SafeCoinsIllustration />
+          </Box>
+          <Box flex={1}>
+            <Typography variant="subtitle2" fontWeight="bold" color="static.main" className={css.title}>
+              Enjoy No Fee November
+            </Typography>
+            <Typography variant="body2" color="static.light" className={css.description}>
+              SAFE holders enjoy gasless transactions on Mainnet this November.{' '}
+              <Link href="https://help.safe.global/en/" style={{ textDecoration: 'underline', fontWeight: 'bold' }}>
+                Learn more
+              </Link>
+            </Typography>
+          </Box>
+        </Stack>
+      </Card>
+    )
+  }
+
+  // Eligible state (from Figma design)
+  if (isEligible === true) {
+    return (
+      <Card className={css.card}>
+        <Stack direction="row" alignItems="center" spacing={3}>
+          <Box className={css.iconContainer}>
+            <SafeCoinsIllustration />
+          </Box>
+          <Box flex={1}>
+            <Typography variant="subtitle2" fontWeight="bold" color="static.main" className={css.title}>
+              Enjoy No Fee November
+            </Typography>
+            <Typography variant="body2" color="static.light" className={css.description}>
+              SAFE holders enjoy gasless transactions on Mainnet this November.{' '}
+              <Link href="https://help.safe.global/en/" style={{ textDecoration: 'underline', fontWeight: 'bold' }}>
+                Learn more
+              </Link>
+            </Typography>
+          </Box>
+        </Stack>
+      </Card>
+    )
+  }
+
+  // Not eligible state (from banner in assets/home, without Get SAFE button)
+  return (
+    <Card className={css.card}>
+      <Stack direction="row" alignItems="center" spacing={3}>
+        <Box className={css.iconContainer}>
+          <SafeCoinsIllustration />
+        </Box>
+        <Box flex={1}>
+          <Typography variant="subtitle2" fontWeight="bold" color="static.main" className={css.title}>
+            Enjoy No Fee November
```

Comment:
It should be spelled `No-Fee` as per English grammar. We might want to update it everywhere.